packages/miniflare/src/workers/queues/schemas.ts

```diff
@@ -1,12 +1,29 @@
 import { Base64DataSchema, z } from "miniflare:zod";
 
+export const QueueMessageDelaySchema = z.number().int().min(0).max(43200).optional();
+
+export const QueueProducerOptionsSchema = /* @__PURE__ */ z.object({
+	// https://developers.cloudflare.com/queues/platform/configuration/#producer
+	queueName: z.string(),
+	deliveryDelay: QueueMessageDelaySchema,
+});
+
+export const QueueProducerSchema = /* @__PURE__ */ z.intersection(
+	QueueProducerOptionsSchema,
+	z.object({ workerName: z.string() })
+);
+export type QueueProducer = z.infer<typeof QueueProducerSchema>;
+export const QueueProducersSchema =
+	/* @__PURE__ */ z.record(QueueProducerSchema);
+
 export const QueueConsumerOptionsSchema = /* @__PURE__ */ z.object({
 	// https://developers.cloudflare.com/queues/platform/configuration/#consumer
 	// https://developers.cloudflare.com/queues/platform/limits/
 	maxBatchSize: z.number().min(0).max(100).optional(),
 	maxBatchTimeout: z.number().min(0).max(30).optional(), // seconds
 	maxRetires: z.number().min(0).max(100).optional(),
 	deadLetterQueue: z.ostring(),
+	retryDelay: QueueMessageDelaySchema,
```

Comment:
Can you update the Miniflare README with this new option?

https://github.com/cloudflare/workers-sdk/blob/main/packages/miniflare/README.md#interface-queueconsumeroptions